clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/CloudMetricController.groovy

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.controllers
+
+import com.netflix.spinnaker.clouddriver.model.CloudMetric
+import com.netflix.spinnaker.clouddriver.model.CloudMetricProvider
+import com.netflix.spinnaker.clouddriver.model.CloudMetricStatistics
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestMethod
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+
+@RequestMapping("/cloudMetrics")
+@RestController
+class CloudMetricController {
+
+  @Autowired
+  List<CloudMetricProvider> metricProviders
+
+  @RequestMapping(method = RequestMethod.GET, value = "/{cloudProvider}/{account}/{region}")
+  List<CloudMetric> findAll(@PathVariable String cloudProvider,
```

Comment:
It's to support Deck, but also to support AWS, since we need to create different API clients per region to query the metrics data. I'm assuming it's still desirable on the GCP as a filter, since you can have multiple server groups with the same name (different identifiers, but API users don't want to know those) in different regions.